client/client-multi/client-cpp/Modules/FindBotan.cmake

```diff
@@ -25,28 +25,31 @@
 
 message("\nLooking for Botan C++ headers and libraries")
 
+set(BOTAN_CODE_VERSION botan-1.11) 
```

Comment:
It tried to use it BOTAN_VERSION before, but it has an error, looks like CMake has it.
I propose use set(BOTAN_VERSION_CODE 1.11)